Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -224,6 +224,9 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
             if requirements.any? { |r| r.is_a?(MacOSRequirement) && !r.version }
               puts "#{Tty.bold}#{Tty.red}#{name}#{Tty.reset}: requires macOS" if any_named_args
               next
+            elsif requirements.any? { |r| r.is_a?(ArchRequirement) && r.arch != @bottle_tag.arch }
+              puts "#{Tty.bold}#{Tty.red}#{name}#{Tty.reset}: doesn't support this Linux" if any_named_args
```

Comment:
I think it would be nice to specify the architecture here, like "foo doesn't support #{arch} Linux".

Reply:
Agreed.